src/fu-main.c

```diff
@@ -341,12 +341,12 @@ fu_main_authorization_is_valid (PolkitAuthorizationResult *auth, GError **error)
 {
 	/* failed */
 	if (auth == NULL) {
-		g_autofree gchar *message = g_strdup ((*error)->message);
+		g_debug ("%s", (*error)->message);
 		g_clear_error (error);
-		g_set_error (error,
-			     FWUPD_ERROR,
-			     FWUPD_ERROR_AUTH_FAILED,
-			     "Could not check for auth: %s", message);
+		g_set_error_literal (error,
```

Comment:
I don't think that's always true; I think we need to check using `g_error_matches()` using the correct domain and code (and the local error should really be `error_local` anyway, rather than clobbering the passed error...)

Reply:
The errors are GDBus errors aren't they?  So you lose the correct domain and code I would think.

Reply:
Not sure what domain they are -- I'd be surprised if they were GDbus ones...

Reply:
The error shows up as:
```
GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: Action org.freedesktop.fwupd.update-internal is not registered
```

Reply:
Looks like it mapped out to domain 329, code 0

Reply:
And yeah, that maps to `POLKIT_ERROR` for me, i'll adjust.